asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -152,10 +160,26 @@ def from_oemol(cls, mol: oechem.OEMol, **kwargs) -> "Ligand":
         }
         kwargs["tags"] = tags
         # clean the sdf data for the internal model
-        sdf_str = oemol_to_sdf_string(clear_SD_data(mol))
+        sdf_str = oemol_to_sdf_string(clear_SD_data(input_mol))
+        # create a smiles which does not have nitrogen stereo
+        smiles = oemol_to_smiles(input_mol).replace("[N@@]", "N").replace("[N@]", "N")
```

Comment:
This is not the smiles exactly as computed by openeye due to the replacement, should the field name reflect this?